File: CS/WindowsApplication1/Class1.cs
using System;

namespace WindowsApplication1 {
	public class Record {
		private int fID;
		private string fProductName;
		private string fCategory;
		private bool fDiscontinued;
		private double fUnitPrice;
        private int fA;
        private int fB;
        private int fC;
        private int fD;
        private int fE;
        private int fF;
        private int fG;
        private int fH;
        private int fI;
        private int fJ;

		public Record(int fID, string fProductName, string fCategory, bool fDiscontinued, double fUnitPrice) {
			this.fID = fID;
			this.fProductName = fProductName;
			this.fCategory = fCategory;
			this.fDiscontinued = fDiscontinued;
			this.fUnitPrice = fUnitPrice;
		}

		public int ID {
			get { return fID; }
		}

		public string ProductName {
			get { return fProductName; }
			set { fProductName = value; }
		}

		public string Category {
			get { return fCategory; }
			set { fCategory = value; }
		}

		public bool Discontinued {
			get { return fDiscontinued; }
			set { fDiscontinued = value; }
		}

		public double UnitPrice {
			get { return fUnitPrice; }
			set { fUnitPrice = value; }
		}

        public int A{
            get { return fA; }
            set { fA = value; }
        }
        public int B{
            get { return fB; }
            set { fB = value; }
        }
        public int C{
            get { return fC; }
            set { fC = value; }
        }
        public int D{
            get { return fD; }
            set { fD = value; }
        }
        public int E{
            get { return fE; }
            set { fE = value; }
        }
        public int F{
            get { return fF; }
            set { fF = value; }
        }

        public int G{
            get { return fG; }
            set { fF = value; }
        }

        public int H{
            get { return fF; }
            set { fF = value; }
        }

        public int I{
            get { return fF; }
            set { fF = value; }
        }
        public int J{
            get { return fF; }
            set { fF = value; }
        }
    }
}

File: VB/WindowsApplication1/Class1.vb
Imports Microsoft.VisualBasic
Imports System

Namespace WindowsApplication1
	Public Class Record
		Private fID As Integer
		Private fProductName As String
		Private fCategory As String
		Private fDiscontinued As Boolean
		Private fUnitPrice As Double
		Private fA As Integer
		Private fB As Integer
		Private fC As Integer
		Private fD As Integer
		Private fE As Integer
		Private fF As Integer
		Private fG As Integer
		Private fH As Integer
		Private fI As Integer
		Private fJ As Integer

		Public Sub New(ByVal fID As Integer, ByVal fProductName As String, ByVal fCategory As String, ByVal fDiscontinued As Boolean, ByVal fUnitPrice As Double)
			Me.fID = fID
			Me.fProductName = fProductName
			Me.fCategory = fCategory
			Me.fDiscontinued = fDiscontinued
			Me.fUnitPrice = fUnitPrice
		End Sub

		Public ReadOnly Property ID() As Integer
			Get
				Return fID
			End Get
		End Property

		Public Property ProductName() As String
			Get
				Return fProductName
			End Get
			Set(ByVal value As String)
				fProductName = value
			End Set
		End Property

		Public Property Category() As String
			Get
				Return fCategory
			End Get
			Set(ByVal value As String)
				fCategory = value
			End Set
		End Property

		Public Property Discontinued() As Boolean
			Get
				Return fDiscontinued
			End Get
			Set(ByVal value As Boolean)
				fDiscontinued = value
			End Set
		End Property

		Public Property UnitPrice() As Double
			Get
				Return fUnitPrice
			End Get
			Set(ByVal value As Double)
				fUnitPrice = value
			End Set
		End Property

		Public Property A() As Integer
			Get
				Return fA
			End Get
			Set(ByVal value As Integer)
				fA = value
			End Set
		End Property
		Public Property B() As Integer
			Get
				Return fB
			End Get
			Set(ByVal value As Integer)
				fB = value
			End Set
		End Property
		Public Property C() As Integer
			Get
				Return fC
			End Get
			Set(ByVal value As Integer)
				fC = value
			End Set
		End Property
		Public Property D() As Integer
			Get
				Return fD
			End Get
			Set(ByVal value As Integer)
				fD = value
			End Set
		End Property
		Public Property E() As Integer
			Get
				Return fE
			End Get
			Set(ByVal value As Integer)
				fE = value
			End Set
		End Property
		Public Property F() As Integer
			Get
				Return fF
			End Get
			Set(ByVal value As Integer)
				fF = value
			End Set
		End Property

		Public Property G() As Integer
			Get
				Return fG
			End Get
			Set(ByVal value As Integer)
				fF = value
			End Set
		End Property

		Public Property H() As Integer
			Get
				Return fF
			End Get
			Set(ByVal value As Integer)
				fF = value
			End Set
		End Property

		Public Property I() As Integer
			Get
				Return fF
			End Get
			Set(ByVal value As Integer)
				fF = value
			End Set
		End Property
		Public Property J() As Integer
			Get
				Return fF
			End Get
			Set(ByVal value As Integer)
				fF = value
			End Set
		End Property
	End Class
End Namespace
